templates/lxc-download.in

```diff
@@ -222,6 +228,7 @@ Optional arguments:
 [ --no-validate ]: Disable GPG validation (not recommended)
 [ --flush-cache ]: Flush the local copy (if present)
 [ --force-cache ]: Force the use of the local copy even if expired
+[ --standard-resolver <resolver> ]: Force the use of a specific standard resolver
```

Comment:
This shouldn't take arguments, it should just be `--standard-resolver` and if passed should set DOWNLOAD_STANDARD_RESOLVER to `true`.